src/components/Popups/PopupDrawingToolbar.tsx

```diff
@@ -50,16 +54,28 @@ const PopupDrawingToolbar = ({
     canRedo,
     canUndo,
     className,
+    drawnPathGroups,
     onDelete,
     onRedo,
     onReply,
     onUndo,
     reference,
 }: Props): JSX.Element => {
     const intl = useIntl();
+    const popupRef = React.useRef<PopupBase>(null);
+    const prevDrawnPathGroups = usePrevious(drawnPathGroups);
+
+    React.useEffect(() => {
+        const { current: popup } = popupRef;
+
+        if (popup?.popper && prevDrawnPathGroups && drawnPathGroups.length !== prevDrawnPathGroups.length) {
```

Comment:
I don't think these checks or the prevDrawnPathGroups state is needed, since this effect will only be run when either `popupRef` or `drawnPathGroups` changes.